test-two/src/index.js

```diff
@@ -4,23 +4,29 @@ import { getUsersForOrganisation } from "./services/github";
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
 const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+var pageNumber;
 
 export const go = async () => {
-  try {
-    const users = await getUsersForOrganisation(
-      baseUrl,
-      gitHubApiAuthToken,
-      organisationId
-    );
 
-    const output = users.reduce((result, user) => {
-      return `${result}Username: ${user.login}\n`;
-    }, "");
+    for (pageNumber = 1; pageNumber <= 23; pageNumber++) {
+        try {
```

Comment:
Is this the ideal place for try catch in this file?

Reply:
No, I should of put the try and catch outside the for loop sorry.